src/engine/strat_engine/blockdev.rs

```diff
@@ -26,19 +26,25 @@ pub struct StratBlockDev {
     pub devnode: PathBuf,
     bda: BDA,
     used: RangeAllocator,
+    location: Option<String>,
+    disk_id: Option<String>,
```

Comment:
Reiterating concern re naming of attributes. ```disk_id``` is actually ```hardware_id``` while ```location``` is actually ```user_id````? Hard to know what is what.